src/components/CreateMenu.js

```diff
@@ -59,7 +88,8 @@ class CreateMenu extends PureComponent {
                     this.onModalHide();
                     this.resetOnModalHide();
                 }}
-                anchorPosition={styles.createMenuPosition}
+                anchorPosition={this.getAnchorPosition(this.props.popOverType)}
```

Comment:
If the anchor position is determined by the popover type, and the popover type is already passed as a prop, then I think it would be cleaner to have anchor position determined inside of the `<Popover>` component. Would this be possible?